.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,70 @@
-**Please provide the following information**
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+
+Be honest!
+-->
+
+- [ ] I've read the **issue reporting guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good report with the required information
+- [ ] I've verified that **the issue is not fixed and is reproducible** in the **latest version, 4.2.3**
```

Comment:
I doubt maintainers are a fan of more things to maintain with versioning.

Why not:
`- [ ] I've verified that **the issue is not fixed and is reproducible** in the [latest version](https://www.qbittorrent.org/download.php)`

Reply:
Thanks, fixed